app/ts/InlineHelp/lexicon.ts

```diff
@@ -1,136 +1,136 @@
-import IHelpInfo from './IHelpInfo';
+import {IHelpInfo} from 'help-popup';
 
 export const lexicon: Record<string, IHelpInfo> = {
   'add-subproblem': {
     title: 'Adding a new problem',
     text:
       'Here you can define a new problem as a subset of all evidence (by excluding criteria and alternatives), and with different scale ranges.',
-    link: '/manual.html#mcda-problem-creation'
+    link: 'mcdaHost/manual.html#mcda-problem-creation'
```

Comment:
prefer to go the other direction and only have anchor here, and move host plus path to `HelpContext` configuration